src/snowcli/app/dev/docs/templates/usage.rst.jinja2

```diff
@@ -7,7 +7,7 @@ Syntax
 
   snow {% for part in path %}{{ part }}{{ " " if not loop.last }}{% endfor %}
 {%- for param in options if not param.hidden %}
-{{ "    " + param.opts[0] }}{% if not param.is_flag %} <{{ param.name }}>{% endif %}
+{{ "    " + (param.opts | sort | first) }}{% if not param.is_flag %} <{{ param.name }}>{% endif %}
```

Comment:
It's clever but I think it may raise issues when we have multiple variants starting with `--` (for example to provide backward compatibility). If the old flag starts with "a" it will be chosen instead of using the new one.